src/lib/docpad.coffee

```diff
@@ -1737,12 +1737,12 @@ class DocPad extends EventEmitterGrouped
 	# Initialization Functions
 
 	###*
-	# Construct DocPad
+	# Constructor method. Creates
+	# the DocPad TaskGroup object.
```

Comment:
Should be something like `Constructor method. Sets up the DocPad instance.`

As DocPad uses TaskGroup, it doesn't inherit or extend TaskGroup.